tls/s2n_server_cert.c

```diff
@@ -43,7 +39,7 @@ int s2n_server_cert_recv(struct s2n_connection *conn)
     cert_chain.data = s2n_stuffer_raw_read(&conn->handshake.io, size_of_all_certificates);
     cert_chain.size = size_of_all_certificates;
 
-    const s2n_cert_validation_code rc = conn->config->verify_cert_chain_cb(conn, cert_chain.data, cert_chain.size, &cert_type, &public_key, conn->config->verify_cert_context);
+    const s2n_cert_validation_code rc = s2n_x509_validator_validate_cert_chain(&conn->x509_validator, conn, cert_chain.data, cert_chain.size, &cert_type, &public_key);
 
     if (rc != S2N_CERT_OK) {
         S2N_ERROR(S2N_ERR_CERT_UNTRUSTED);
```

Comment:
Could replace with the new S2N_ERROR_IF macro

Reply:
I also opened an issue for refactoring the rest of the code base here: https://github.com/awslabs/s2n/issues/692

Reply:
I can't find that macro anywhere in the code base. What header file is it supposed to be in?